src/components/CardSelection/CardSelection.tsx

```diff
@@ -53,9 +53,11 @@ const CardSelectionItem = <T, >({
     >
       <div className={classnames(styles.content, isHorizontal && styles.horizontal)}>
         <div className={styles.header}>
-          {
-            icon && <Icon component={icon} size={isHorizontal ? 24 : 32} />
-          }
+          {icon && (
+            <div className={classnames(styles.icon, isHorizontal && styles.horizontalIcon)}>
```

Comment:
`isHorizontal && styles.horizontalIcon` will add a `false` class if isHorizontal is falsey. I'd use a ternary operator to produce cleaner classes (unless classnames discards non-string values)

Reply:
I guess `classnames` ignores falsy values!

https://www.npmjs.com/package/classnames#usage